spec/controllers/post_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /posts #index' do
+    subject { get '/users/1/posts' }
+    before { subject }
+
+    it 'responds successfully with an HTTP 200 status code' do
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the index template' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'has the correct placeholder text' do
+      expect(response.body).to include('posts')
+    end
+  end
+
+  describe 'GET /posts #show' do
+    subject { get '/users/1/posts/1' }
+    before { subject }
+
+    it 'responds successfully with an HTTP 200 status code' do
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the show template' do
+      expect(response).to render_template(:show)
+    end
+
+    it 'has the correct placeholder text' do
+      expect(response.body).to include('post')
+    end
```

Comment:
- [ ] Great job do far but I think this tests are NOT passed since after running rspec spec/controllers it shows me this error so kindly let your tests all passed as required ✅✨♻🔥💯💯

![image](https://user-images.githubusercontent.com/58553711/221729833-fa2ba992-e78f-46f5-840c-9ee7eaf498df.png)

## Here is the error messages to check

![image](https://user-images.githubusercontent.com/58553711/221730236-ba48722e-6e08-4f12-99b4-63655521587c.png)